PCbuild/build.bat

```diff
@@ -115,7 +115,8 @@ if exist "%GIT%" set GITProperty=/p:GIT="%GIT%"
 if not exist "%GIT%" echo Cannot find Git on PATH & set GITProperty=
 
 rem Setup the environment
-call "%dir%env.bat" %vs_platf% >nul
```

Comment:
Looks like there are some missing environment pieces, but I can add those into `python.bat` on build. Looks like that's how we launch for PGO build anyway.